linux_os/guide/system/selinux/selinux_confinement_of_daemons/rule.yml

```
@@ -37,6 +37,13 @@ references:
     iso27001-2013: A.10.1.1,A.11.1.4,A.11.1.5,A.11.2.1,A.12.1.2,A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.5.1,A.12.6.2,A.12.7.1,A.13.1.1,A.13.1.3,A.13.2.1,A.13.2.3,A.13.2.4,A.14.1.2,A.14.1.3,A.14.2.2,A.14.2.3,A.14.2.4,A.6.1.2,A.7.1.1,A.7.1.2,A.7.3.1,A.8.2.2,A.8.2.3,A.9.1.1,A.9.1.2,A.9.2.3,A.9.4.1,A.9.4.4,A.9.4.5
     cis-csc: 1,11,12,13,14,15,16,18,3,5,6,9
 
+ocil_clause: 'SELinux policy contains rules for all the daemons running on the system'
```

The ocil clause should be a sentence describing failing the check.
```suggestion
ocil_clause: 'There are unconfined daemons running on the system'
```